layer/src/main/scala/geotrellis/layer/mapalgebra/focal/FocalTileLayerCollectionMethods.scala

```diff
@@ -18,7 +18,7 @@ package geotrellis.layer.mapalgebra.focal
 
 import geotrellis.raster._
 import geotrellis.raster.Neighborhoods.Square
-import geotrellis.raster.mapalgebra.focal.{Kernel, Aspect, Sum, Max, Min, Mean, Median, Mode, StandardDeviation, Convolve, Slope}
+import geotrellis.raster.mapalgebra.focal.{TargetCell => TC, _}
```

Comment:
Why do we need this import change here?

Reply:
Oh, that was something I meant to change back. I will do that now :+1: